spec/01-lexical-syntax.md

```diff
@@ -344,10 +344,12 @@ Literal  ::=  [‘-’] integerLiteral
 ### Integer Literals
 
 ```ebnf
-integerLiteral  ::=  (decimalNumeral | hexNumeral)
+integerLiteral  ::=  (decimalNumeral | hexNumeral | binaryNumberal)
                        [‘L’ | ‘l’]
 decimalNumeral  ::=  digit {digit}
 hexNumeral      ::=  ‘0’ (‘x’ | ‘X’) hexDigit {hexDigit}
+binaryNumeral   ::=  ‘0’ (‘b’ | ‘B’) binaryDigit {binaryDigit}
+binaryDigit     ::=  ‘0’ | ‘1’
```

Comment:
```suggestion
```
Since there is no reminder of hexDigit, there probably shouldn't be one for binaryDigit

Reply:
I noticed digit was removed (because it's a char class?); and hexDigit was promoted to earlier section because of unicode escape. But maybe the rule of thumb should be "this is obvious so defer to the summary" as opposed to "chapter summary should be complete".

Reply:
I think that is a good rule of thumb

Reply:
thanks, I took both edits. The earlier hexdigit might be unnecessary, but the context is (TIL):
```
scala> val `\u0061\u0062\u0063` = 42
val abc: Int = 42
```

Reply:
Yeah I learned that as well a couple days ago, it's a cool feature in case you are restricted to ASCII, but still need to access some unicode-named methods